components/server/src/workspace/context-parser.ts

```diff
@@ -96,12 +96,24 @@ export interface URLParts {
  * and expects "othercontext" to be parsed and passed back.
  */
 export interface IPrefixContextParser {
+    // readonly property to see if prefix context will specify an IDE
```

Comment:
Update interface `IPrefixContextParser` so that other prefix context parsers that may be added in the future will not break this PR